typescript/nuxtjs-prisma-nuxt-module/server/api/user-exists.ts

```diff
@@ -0,0 +1,9 @@
+import prisma from '~/lib/prisma'
+
+export default eventHandler(async () => {
+  const user = await prisma.user.exists({
+    email: 'niko@prisma.io',
+  })
```

Comment:
```suggestion
  const user = await prisma.user.findFirst({
    where: { email: 'niko@prisma.io' },
  })
```

Reply:
I'm keeping the `exists` example to educate our users on how the usage of a custom prisma client would look like.